src/sidebar/components/MenuArrow.tsx

```diff
@@ -4,23 +4,23 @@ import {
 } from '@hypothesis/frontend-shared/lib/next';
 import classnames from 'classnames';
 
-/**
- * @typedef MenuArrowProps
- * @prop {string} [classes]
- * @prop {'up'|'down'} [direction]
- */
+type MenuArrowProps = {
+  classes?: string;
+  direction?: 'up' | 'down';
```

Comment:
I've seen there are a couple of other places where there's a type defined as `'up' | 'down'`.

Should we extract a type definition, let's say `type VerticalDirection = 'up' | 'down'`, that we can then reuse? If so, where would be the best place?

Reply:
I'm not sure if the abstraction is a useful shareable one (i.e. if it lives externally, it might add a layer of indirection that makes the context more distributed than it need be).

One option I've been using lately is `enum`; that might work here. But I'd probably keep it within this module and not try to generalize it.